cmd/explaintest/r/imdbload.result

```diff
@@ -279,6 +279,15 @@ id	estRows	task	access object	operator info
 TableReader_7	804024.75	root		data:Selection_6
 └─Selection_6	804024.75	cop[tikv]		or(and(eq(imdbload.char_name.imdb_index, "I"), lt(imdbload.char_name.surname_pcode, "E436")), and(eq(imdbload.char_name.imdb_index, "L"), lt(imdbload.char_name.surname_pcode, "E436")))
   └─TableFullScan_5	4314864.00	cop[tikv]	table:char_name	keep order:false
+explain select * from char_name use index (itest2) where ((imdb_index = 'I') and (surname_pcode < 'E436')) or ((imdb_index = 'L') and (surname_pcode < 'E436'));
```

Comment:
Changes here are irrelevant to this PR. My next PR will need these test results for comparison.